web/src/js/components/flowview/index.js

```diff
@@ -89,7 +89,6 @@ var FlowView = React.createClass({
             } else {
                 active = tabs[0];
             }
-            this.selectTab(active);
```

Comment:
I'd like to keep `render` pure.
Also, it leads to an issue when `selectedFlow` is not sync with `routerParams.flowId`